src/VersionTool/Repository.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace VersionTool
+{
+    public class Repository
+    {
+        public Repository(string name, string path)
+        {
+            Name = name;
+            Path = path;
+        }
+
+        public string Name { get; }
+        public string Path { get; }
```

Comment:
You might be able to stick `[JsonIgnore]` on these properties and avoid a custom converter.